src/Makefile.am

```diff
@@ -2746,3 +2746,56 @@ tests_testEventLoop_LDADD = \
 	$(COMPAT_LIB) \
 	$(XTRA_LIBS)
 tests_testEventLoop_LDFLAGS = $(LIBADD_DL)
+
+if ENABLE_FUZZ_DECODE
+EXTRA_PROGRAMS =+ fuzzHttp1Parser fuzzRFC1035
+
+fuzzHttp1Parser_SOURCES = \
```

Comment:
We already have extensive edge case testing for the `Http1::Parser` class. I do not believe fuzzer adds any benefit here.

Reply:
Well, this PR primarily aims to integrate `squid-cache` into the [oss-fuzz](https://google.github.io/oss-fuzz) infrastructure for continuous fuzzing.

Reply:
> We already have extensive edge case testing for the Http1::Parser class. I do not believe fuzzer adds any benefit here.

FWIW, I disagree that testing Http1::Parser using fuzzers adds no benefit.

Reply:
What benefits exactly/specifically do you see us getting from fuzzing the `Http1::Parser` class?
 This is a class where the edge cases are small for each syntax token and well-known.

IME fuzzing is most useful for smoke testing larger integrated systems - such as caching, HTTP relay. The level where unit tests become overly large and burdensome to write a unique test for each and every edge case. A fuzzer test alongside Daft tests, CoAdvisor and performance measurements would be good to add IMO.